Source/WebCore/html/RadioInputType.cpp

```diff
@@ -96,11 +97,11 @@ void RadioInputType::willUpdateCheckedness(bool nowChecked, WasSetByJavaScript)
 {
     if (!nowChecked)
         return;
-    if (element()->radioButtonGroups()) {
```

Comment:
Seems we should have a local variable for the element to avoid calling `protectedElement()` twice and causing unnecessary refcounting churn.